### PR TITLE
APM-81: APM Convert Tokens process plugin created.

### PR DESCRIPTION
**Pull request by pranitjha**

### CODE REVIEW

File: src/Plugin/migrate/process/ApmConvertTokens.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * )
+ */
+class ApmConvertTokens extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Token mapping detail from configuration file.
+    $tokens_mapping = $this->configuration['map'];
```
Comment (joshirohit100) [inline src/Plugin/migrate/process/ApmConvertTokens.php]: check for this if required

Comment (joshirohit100) [reply src/Plugin/migrate/process/ApmConvertTokens.php]: what I meant was check if `$this->configuration['map']` available or not - user has added `map` key in yml or not. If not, then not process and return the same un-altered value.

Comment (pranitjha) [reply src/Plugin/migrate/process/ApmConvertTokens.php]: Added condition to check $this->configuration['map'] is not empty.

**Comment by joshirohit100** [comment]
PR looks fine to me

cc @amitgoyal  for review and merge.

**Pull request merged**